src/app/api/collections/models.py

```diff
@@ -13,3 +13,11 @@ class CollectionNode(BaseModel):
     title: Optional[str]  # might be none due to data model
     children: list[CollectionNode]
     parent_id: Optional[UUID]
+
+
+class MissingMaterials(CollectionNode):
```

Comment:
some documentation what the actual attributes are would be nice :P

Reply:
Added docstring and comments